jekyll/_cci2/security.md

```diff
@@ -119,6 +104,19 @@ Following are the system events that are logged. See `action` in the Field secti
 - **success:** A flag to indicate if the action was successful.
 - **request:** If this event was triggered by an external request this data will be populated and may be used to connect events that originate from the same external request. The format is a JSON blob containing `id` (the request ID assigned to this request by CircleCI), `ip_address` (the original IP address in IPV4 dotted notation from which the request was made, eg. 127.0.0.1), and `client_trace_id` (the client trace ID header, if present, from the 'X-Client-Trace-Id' HTTP header of the original request).
 
+## Checklist To Using CircleCI Securely as a Customer
+
+If you are getting started with CircleCI there are some things you can ask your team to consider for security best practices as _users_ of CircleCI:
+
+- Minimise the number of secrets (private keys / environment variables) your build needs.
```

Comment:
I think it would be good to add another item in here about regularly rotating secrets - although currently this is a bit of a pain to automate if you use contexts, as there's no official API.